src/Text/Layout/Table/Spec/Position.hs

```diff
@@ -23,8 +23,13 @@ instance Show (Position V) where
         Center -> "center"
         End    -> "bottom"
 
+-- | The default 'Position' displays at the left or top, depending on the orientation.
 instance Default (Position orientation) where
-    def = Start
+    def = startPosition
+
+-- | Displays at the left or top, depending on the orientation.
+startPosition :: Position orientation
+startPosition = Start
```

Comment:
The name feels a bit alien considering that the other ones are all single words without _position_ in them.  Any idea how we could make it more consistent?

Reply:
Agreed. I originally went with `startPosition` because I assumed that `start` would be too common and would conflict with other bindings. That said, I don't see too many of them on hoogle, so `start` might not be terrible.

Other ideas:
- begin
- beginning
- topLeft

Reply:
In the worst case one can use qualified imports.  Feel free to choose one:
* `start`
* `beginning` (I think it should be a noun.)
* `topOrLeft` (If you like neither of the former ones.  Reflects the polymorphic nature.)

Reply:
I went with beginning.